openquake/calculators/export/risk.py

```diff
@@ -437,16 +437,32 @@ def export_damages_csv(ekey, dstore):
     return writer.getsaved()
 
 
+def modal_damage_array(data, damage_dt):
+    # determine the damage state with the highest probability
+    A, L, MS, D = data.shape
+    dmgstate = damage_dt['structural'].names
```

Comment:
For calculations where `number_of_ground_motion_fields > 1`,

`dmgstate` will contain something like ('no_damage_mean', 'no_damage_stdv', 'slight_mean', 'slight_stdv', 'moderate_mean', 'moderate_stdv', 'extreme_mean', 'extreme_stdv', 'complete_mean', 'complete_stdv'). 

Instead, it should be the names of the damage states, i.e., ('no_damage', 'slight', 'moderate', 'extreme', 'complete').